pkg/crc/machine/kubeconfig.go

```diff
@@ -0,0 +1,109 @@
+package machine
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+)
+
+func WriteKubeconfig(config ClusterConfig) error {
+	dir, err := ioutil.TempDir("", "crc")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(dir)
+	dev, err := writeContext(dir, config, "developer", "developer", "crc-dev")
+	if err != nil {
+		return err
+	}
+	admin, err := writeContext(dir, config, "kubeadmin", config.KubeAdminPass, "crc-admin")
+	if err != nil {
+		return err
+	}
+
+	rootConfig := filepath.Join(constants.GetHomeDir(), ".kube", "config")
```

Comment:
To simplify, we can do:
* If KUBECONFIG is set, print a warning and do nothing. `You need to login, etc.`
* otherwise use the default file.

I believe that if KUBECONFIG is set, it will be really hard to not mess up with the configuration.

Reply:
@guillaumerose in minishift we used to do that without any issue. But there we used to parse the kubeconfig file instead using the oc commands (that doesn't need to be done here) . 

- Check if KUBECONFIG set the use this ase rootConfig (if file doesn't exist then create, I think same we are going to do with `~/.kube/config` file )
- otherwise default file `~/.kube/config` 

Because I am sure if we make this change the next request would be honoring the KUBECONFIG parameter.

Reply:
I change the logic. Let me know if it's better.